layouts/page/pulumi-up.html

```diff
@@ -1,39 +1,105 @@
 {{ define "main" }}
-    <div class="pulumi-up-body text-white py-12">
-        <div class="container mx-auto">
-            <section id="about">
+    <div class="pulumi-up-body text-white pb-12 ">
+        <div class="relative overflow-hidden pulumi-up-video-section">
+            <video class="pulumiup-video" autoplay muted loop>
+                <source src="/video/pulumiup-lofi.mp4" type="video/mp4">
```

Comment:
It'd be cool to do this as CSS+SVG, since it's all just flat shapes.

Reply:
That `.mp4` is the only asset I got. Aaron worked with a contractor I believe, so he will probably be able to help you out!